src/main/java/org/fundacionjala/coding/jorge/FizzBuzz.java

```diff
@@ -0,0 +1,42 @@
+package org.fundacionjala.coding.jorge;
+
+import java.util.stream.IntStream;
+
+/**
+ *
+ */
+public final class FizzBuzz {
+
+    /**
+     * The first part.
+     *
+     * @param rango int[]
+     * @return Object[]
+     */
+    public Object[] fizzBuzz(final int[] rango) {
+        return IntStream.rangeClosed(rango[0], rango[1]).mapToObj(FizzBuzz::changeFB1).toArray();
+
+    }
+
+
+    /**
+     * @param arrayObj to evaluate 3 or 5.
+     * @return return the arrayObj.
+     */
+    public static Object changeFB1(final Object arrayObj) {
+
+        final int fizz = 3;
+        final int buzz = 5;
+        final int fizzBuzz = fizz * buzz;
+        if ((Integer) arrayObj % fizzBuzz == 0) {
+            return "FizzBuzz";
+        }
+        if ((Integer) arrayObj % buzz == 0 || String.valueOf(arrayObj).contains("5")) {
+            return "Buzz";
+        }
+        if ((Integer) arrayObj % fizz == 0 || String.valueOf(arrayObj).contains("3")) {
+            return "Fizz";
+        }
+        return arrayObj;
```

Comment:
This may be refactored like this.
```java
    private static String changeFB1(final int number) {
        final int fizz = 3;
        final int buzz = 5;
        final int fizzBuzz = fizz * buzz;
        if (number % fizzBuzz == 0) {
            return "FizzBuzz";
        }
        final String numberAsString = String.valueOf(number);
        if (number % buzz == 0 || numberAsString.contains("5")) {
            return "Buzz";
        }
        if (number % fizz == 0 || numberAsString.contains("3")) {
            return "Fizz";
        }
        return numberAsString;
    }
```